src/index.js

```diff
@@ -378,7 +379,7 @@ window["StatsigSidecar"] = window["StatsigSidecar"] || {
       this._statsigInstance  = new StatsigClient(apiKey, user, options);
       await this._statsigInstance.initializeAsync();
       runStatsigAutoCapture(this._statsigInstance);
-      
+      if (window?.statsigSessionReplayOptions) runStatsigSessionReplay(this._statsigInstance, window?.statsigSessionReplayOptions);
```

Comment:
If we declare `window.statsigSessionReplayOptions` then install session-replay and pass the options.